worker/categorize.py

```diff
@@ -395,7 +395,7 @@ def _is_viable_candidate(self, phrase):
         words = phrase.split()
 
         # Ignore stop words
-        if words[0].lower() in STOP_WORDS:
+        if len(words) == 0 or words[0].lower() in STOP_WORDS:
```

Comment:
Ok. So if len(words) was 0. This means phrase.split() returned []. This can happen when phrase is " ". However, that wouldn't make it past the < 2 check at the beginning of the function. What about "  " (two spaces)? That would also return []. Same with any number of spaces. Maybe we should call strip() to start the function? Then the len(phrase) < 2 will return false for us?